nuxeo-services/nuxeo-platform-filemanager-core/src/test/java/org/nuxeo/ecm/platform/filemanager/TestFileManagerService.java

```diff
@@ -108,6 +108,14 @@ public void testDefaultCreateFromBlob() throws Exception {
         assertEquals("hello.doc", doc.getProperty("dublincore", "title"));
         assertEquals("hello.doc", doc.getProperty("file", "filename"));
         assertNotNull(doc.getProperty("file", "content"));
+
+        // let's make the same test but this time without mime-type checking
+        doc = service.createDocumentFromBlob(coreSession, input, workspace.getPathAsString(), true,
+            "test-data/hello2.doc", true);
+        assertNotNull(doc);
+        assertEquals("hello2.doc", doc.getProperty("dublincore", "title"));
+        assertEquals("hello2.doc", doc.getProperty("file", "filename"));
+        assertNotNull(doc.getProperty("file", "content"));
```

Comment:
The main goal of this tests is to show that, for a blob with a mime-type that matches the file's name, using the mime-type check on or off should not make a difference. I will improve the test and add another one to make this more clear.